### ISSUE TITLE
Potential reduntacies in GPAD exports

### ISSUE THREAD

**Issue by pgarmiri** [geneontology/noctua#590]
Hi,
We came across a case that 2 sets of 3 identical annotations were generated based on the same publication (PMID:27458189) but with different ECO codes (2 mapping to IDA and one  EXP). (https://www.ebi.ac.uk/QuickGO/annotations?reference=PMID:27458189,27458189)
They were created by SYNX in Noctua and exported in the GPAD file.
Is that a normal behavior? That seems a bit redundant as we don't anotate based on individual experiments, as far I know. Could you please have a look?
Thanks,
Penelope

**Comment by balhoff** [comment]
When there are multiple ECO codes, according to the spec we can only have one on each GPAD line. Would you want a criterion applied to prefer one over another? Thanks!

**Comment by balhoff** [comment]
I think this is the model: http://noctua.geneontology.org/editor/graph/gomodel:SYNGO_2270

**Comment by pgaudet** [comment]
Possibly parent/child ECO (ie EXP/IDA) could be considered the same for these cases ?

**Comment by ukemi** [comment]
I think the spec as it stands is correct as long as the evidence assigned refers to two separate pieces of evidence that support the annotation. Perhaps this is more an issue with ECO where a single experiment can legitimately be supported by more than one code. This should be discussed wrt standard annotation protocol.

**Comment by pgaudet** [comment]
But if EXP and IDA are being used, you cannot tell whether the same evidence was used, can you?

**Comment by ukemi** [comment]
At the level of GPAD, the evidence is explicit, so both were used, right?

**Comment by pgaudet** [comment]
What I mean is that I think SynGO annotates by EXP rather than using more specific codes. 
So it's possible that they have annotated the same figure/experiment that another curator annotated. 

How could we tell ?

**Comment by ukemi** [comment]
We can't unless we track the curator. But isn't that always the case? If Harold and I make the same annotation now I'm not sure we would catch it.

**Comment by vanaukenk** [comment]
I've just started QCing the SynGO annotations we are getting the the WormBase file and I think there are cases where they are actually using multiple, more granular evidence codes for the same experiment that sometimes map up to a code like IDA and sometimes map up to EXP.  
I'd need to investigate more thoroughly to be sure, but perhaps this explains what we are seeing?

**Comment by ukemi** [comment]
Yes, I suspect it is. See my comment from 22 hours ago.

**Comment by pgarmiri** [comment]
Yes, this is exactly what we saw. The question was whether we are capturing and annotating every single experiment from a paper supporting the same annotation many times or we do one annotation with the best evidence.
For traditional curation, we were told not to make the same annotation with IDA and IMP with support from the same paper, in order to avoid redundancy.
